src/lib/db/types.ts

```diff
@@ -70,6 +70,7 @@ export type TagDoc = {
 } & TagDocEditibleProps
 
 export type TagDocEditibleProps = {
+  color?: string
   data: JsonObject
```

Comment:
I'd like to put it in `data` like `bookmarked` prop of `NoteDoc`

Reply:
It's a bit more work, because it is not a boolean.
But I think I managed to do it.